pkgs/development/python-modules/minc2-simple/default.nix

```
@@ -0,0 +1,28 @@
+{ buildPythonPackage, fetchFromGitHub, stdenv, cmake,
+  hdf5, libminc, minc2_simple, minc_tools, mni_autoreg,
+  cffi, six, numpy, scipy, pytest }:
+
+buildPythonPackage rec {
+  name     = "minc2-simple-${version}";
+  version  = "2018-12-20";
+
+  src = fetchFromGitHub {
```

instead of having two locations to bump the src, you could just inherit this from minc2_simple
```suggestion
  src = minc2_simple.src;
```